src/funding/status-card/status-card.ts

```diff
@@ -0,0 +1,33 @@
+import "./status-card.scss";
+import { containerless, bindable } from "aurelia-framework";
+import { IDAO } from "entities/DealRegistrationTokenSwap";
+import { ITokenFunding } from "entities/TokenFunding";
+import { DealStatus } from "entities/IDealTypes";
+
+@containerless
+export class StatusCard {
+  @bindable dao: IDAO;
+  @bindable swapCompleted: boolean;
+  @bindable fundingFailed: boolean;
+  private dealStatuses = DealStatus; //have to assign this to a view model field for the HTML to be able to compare enums
+  get chipColor(): string{
```

Comment:
Needs @computedFrom

Reply:
Good catch, thanks @hiaux0 ! I am used to Aurelia 2 where it doesn't need @computedfrom so sometimes I forget to put it on the getters. Appreciate the reminder!